cmd/util/ledger/migrations/cadence_values_migration.go

```diff
@@ -451,14 +458,16 @@ func (t *cadenceValueMigrationReporter) Error(err error) {
 		message = fmt.Sprintf("%s\n%s", message, migrationErr.Stack)
 	}
 
-	t.log.Error().Msgf(
-		"failed to run %s in account %s, domain %s, key %s: %s",
-		migration,
-		storageKey.Address,
-		storageKey.Key,
-		storageMapKey,
-		message,
-	)
+	if t.verboseErrorOutput {
+		t.log.Error().Msgf(
```

Comment:
This is potentially "dangerous", as it may look like that some accounts/values got successfully migrated even though errors occurred, but they got skipped by the sampling